site/hxapi/insurance/confirmation.md

```diff
@@ -123,6 +123,5 @@ Address[]=1 Test St&Address[]=Testchester&Address[]=Testshire&CardHolder=Mr T Te
       <token>000015778</token>
     </Request>
     </API_Header>
-</API_Reply></code>
-
-
+</API_Reply>
```

Comment:
The starting point for this block still has `<code xml>` rather than the triple back tick